Dockerfile

```diff
@@ -22,4 +22,6 @@ WORKDIR /app
 EXPOSE 80
 EXPOSE 443
 COPY --from=publish /app/publish .
+ARG version=unknown
+RUN echo $version > /app/version.txt
```

Comment:
I think that accessing to `version.txt` is a great thing and very useful.

By the moment your application is not ready, so, what do you think about adding a TODO comment?

```suggestion
# TODO: configure static files in the service and copy version.txt to the right folder
RUN echo $version > /app/version.txt
```